app/editor/src/features/content/list-view/ContentFilter.tsx

```diff
@@ -139,6 +139,7 @@ export const ContentFilter: React.FC<IContentFilterProps> = ({ search }) => {
                     ...filter,
                     pageIndex: 0,
                     contentType: e.target.checked ? ContentTypeName.PrintContent : undefined,
+                    printContent: e.target.checked,
```

Comment:
The filter shouldn't have a `printContent` property anymore.  The `contentType==ContentTypeName.PrintContent` is the actual filter.  We don't want to properties to do the same thing.

Reply:
Makes sense, just added it in because `value` and `checked` were relying on it. Will adjust those to `contentType==ContentTypeName.PrintContent`